pages/_language/events/_event/_chapter.vue

```diff
@@ -60,11 +87,12 @@
         params,
       };
     },
+    methods: {
+      smoothScroll (slug) {
+        const element = this.$refs[slug][0];
```

Comment:
easier to use `scrollIntoView`? https://developer.mozilla.org/en-US/docs/Web/API/Element/scrollIntoView

Reply:
This is not supported in Safari using the 'smooth' behavior option.